ksa_compliance/standard_doctypes/payment_entry.py

```diff
@@ -148,3 +150,105 @@ def add_tax_gl_entries(doc, method):
     )
     gl_entries = process_gl_map(gl_entries)
     make_gl_entries(gl_entries)
+
+
+@frappe.whitelist()
+def return_advance_payment_entry_doc(payment_entry, return_amount):
+    payment_entry = json.loads(payment_entry)
+
+    payment_entry_doc = frappe.get_doc("Payment Entry", payment_entry.get("name"))
+    settings = ZATCABusinessSettings.for_company(payment_entry_doc.company)
+    tax = get_taxes_and_charges(payment_entry_doc).taxes[0]
+    tax_rate = tax.rate
+    amount = flt(return_amount)
```

Comment:
Do a check in the backend for the return amount should be no more than the unallocated